tests/testthat/test-get_power.R

```diff
@@ -7,11 +7,11 @@ vcr::use_cassette("daily_ag_point", {
       community = "ag",
       lonlat = c(-179.5, -89.5),
       pars = c("T2M",
-               "T2M_MIN",
-               "T2M_MAX",
-               "RH2M",
-               "WS10M",
-               "PS"),
+        "T2M_MIN",
```

Comment:
oops I used Ctrl+I to fix indenting in RStudio IDE and I see it might have done too much.